static/app/views/starfish/utils/sqlish/SQLishFormatter.spec.tsx

```diff
@@ -21,7 +21,13 @@ describe('SQLishFormatter', function () {
     it('Adds newlines for keywords in INSERTs', () => {
       expect(
         formatter.toString('INSERT INTO users (id, name) VALUES (:c0, :c1) RETURNING *')
-      ).toEqual('INSERT INTO users (id, name) \nVALUES (:c0, :c1) \nRETURNING *');
+      ).toEqual('INSERT INTO users (id, name) \nVALUES (\n  :c0, :c1\n) \nRETURNING *');
+    });
+
+    it('Adds indentation for keywords followed by parentheses', () => {
+      expect(formatter.toString('SELECT * FROM (SELECT * FROM users))')).toEqual(
+        'SELECT * \nFROM (\n  SELECT * \n  FROM users\n))'
```

Comment:
1. In short, not really, but maybe! `toSnapshot()` is only for DOM, it has to be renderable in JSDOM. I'd need to wrap this in an element, or so something else awkward. `toMatchInlineSnapshot()` is even worse because it's subject to Prettier, which reformats the line endings, which is annoying
2. Not mad about adding more test coverage! What are you looking for, though? This is already a nested `SELECT`

Reply:
sorry I meant a double nested subquery (so two nested selects) like you had in your example, just to make sure the indentationLevels lookup works properly.

We can't use `toMatchSnapshot`? It should work for strings.

`expect(formatter.toString(BLAH)).toMatchSnapshot()`

Reply:
No, we can't use `toMatchSnapshot` unfortunately, it's only for renderable DOM. I ended up switching to `toMatchInlineSnapshot()` which I actually prefer because I want to see the resulting formatting in the same file.

I also added a spec for a more complicated subquery condition 👍🏻